pkg/types/types.go

```diff
@@ -1,7 +1,25 @@
 package types
 
+import "fmt"
+
 type Library struct {
+	Id      string
 	Name    string
 	Version string
 	License string `json:",omitempty"`
 }
+
+type Dependency struct {
+	Id        string
```

Comment:
nit: I prefer ID.

```suggestion
	ID        string
```

Reply:
I thought it is camelcase in `go`.

Reply:
You can see here
https://github.com/golang/go/wiki/CodeReviewComments#initialisms